src/main/java/seedu/cafectrl/data/Pantry.java

```diff
@@ -12,15 +12,19 @@ public class Pantry {
     private ArrayList<Dish> menuItems;
     private Ui ui;
 
+    //@@author NaychiMin
     public Pantry(Ui ui, ArrayList<Ingredient> pantryStock) {
         this.ui = ui;
         this.pantryStock = pantryStock;
     }
+    //@@author NaychiMin
 
+    //@@author ShaniceTang
     public Pantry(Ui ui) {
         this.ui = ui;
-        this.pantryStock = new ArrayList<Ingredient>();
+        this.pantryStock = new ArrayList<>();
     }
+    //@@author ShaniceTang

```

Comment:
If you are planning to only chope the Pantry constructor remove ur GitHub tag for the bottom one

here's an explanation found on the CS2113 website:
![image](https://github.com/AY2324S1-CS2113-T17-2/tp/assets/35828587/7ee5f9f4-7da9-4a87-8e9b-6b49d6565f2c)


```suggestion
    //@@author
```